charts/osm/values.yaml

```diff
@@ -131,6 +131,15 @@ OpenServiceMesh:
     # -- Destination's API or collector endpoint where the spans will be sent to
     endpoint: "/api/v2/spans"
 
+  # Inbound external authorization
```

Comment:
This comment should probably be a bit more verbose.

Reply:
will do